app/src/apps/Examinations/lib/questions/MultipleChoiceQuestion.js

```diff
@@ -2,32 +2,53 @@ import { Question } from "../Question";
 
 class MultipleChoiceQuestion extends Question {
   selectedOptionIndex = -1;
+  options;
 
   // TODO: update foo to common question properties once Question contructor updated
-  constructor(foo, options) {
+  constructor(b, n, o) {
     // TODO: init options as MultipleChoiceOption once that class exists
-    super(foo);
+    super(b, n);
+    // add MultipleChoiceOptions
+    o.forEach((option) => {
+      this.addMultipleChoiceOption(
+        option.body,
+        option.correct,
+        option.letter,
+        option.index
+      );
+    });
+  }
+
+  // construct and add new MultipleChoiceOption
+  addMultipleChoiceOption(b, c, l, i) {
+    this.options.push(new MultipleChoiceOption(b, c, l, i));
   }
```

Comment:
This might be fine in this context, but I'd personally find it hard to read this code with the single-letter variable names. I'd need to look at `MultipleChoiceOption` to know what it was expecting. Might be worth adding more? I'll defer to @MathyouMB though.

Reply:
@AngelOnFira yes, I agree. I can definitely add more descriptive variable names. I will wait for @MathyouMB and then adjust it accordingly.